values-schema.yaml

```diff
@@ -274,6 +365,10 @@ definitions:
         type: number
       minReplicas:
         type: number
+  script:
+    default: echo "Hello world"
```

Comment:
It should not have a default value as container image can have default executable command that we are not aware of.